src/tests/orgs.test.js

```diff
@@ -0,0 +1,226 @@
+  return await request(server)
+    .get(ORGS_API_URL + '/:id')
+}
+
+async function updateOrganization(){
+  return await request(server)
+    .put(ORGS_API_URL + '/:id')
+    .send(organization);
+}
+
+async function deleteOrganization(){
+  return await request(server)
+    .delete(ORGS_API_URL + '/:id')
+    .send(organization)
+}
+
+async function removeOrganization(){
+  return await request(server)
+    .delete(ORGS_API_URL + '/:id/remove')
+    .send(organization)
+}
+
+describe('ORG ROUTER', () => {
+  describe('GET ROUTE /', () => {
+    it('should return 200 on success', async () => {
+      await getOrganizations();
+      const res = await request(server)
+        .get(ORGS_API_URL)
+      expect(res.status).toEqual(200);
+    })
+    //should this be 500 or 404 (no users?)
+    it('should return 500 on fail (cannot access database)', async () => {
+      await getOrganizations();
+      const res = await request(server)
+        .get(ORGS_API_URL)
+      expect(res.status).toEqual(500);
+    })
+  })
+
+  describe('POST ROUTE /', () => {
+    it('should return 201 on success', async () => {
+      await insertOrganization();
+      const res = await request(server)
+        .post(ORGS_API_URL)
+        //check this structure
+        .send({
+          name: organization.name,
+          logo: organization.logo,
+          admin_id: 0,
+        })
+      expect(res.status).toEqual(201);
+    })
+    it('should return the new Org on success', async () => {
+      await insertOrganization();
+      const res = await request(server)
+        .post(ORGS_API_URL)
+        .send({
+          name: organization.name,
+          logo: organization.logo,
+          admin_id: 0,
+        })
+      expect(res.body).toEqual({ newOrganization });
+    })
+    it('should return 400 on fail (no org name)', async () => {
+      await insertOrganization();
+      const res = await request(server)
+        .post(ORGS_API_URL)
+        .send({
+          name: '',
+          logo: organization.logo,
+          admin_id: 0,
+        })
+      expect(res.status).toEqual(400);
+    })
+    it('should return 500 on db access fail', async () => {
+      await insertOrganization();
+      const res = await request(server)
+        .post(ORGS_API_URL)
+        .send({
+          name: organization.name,
+          logo: organization.logo,
+          admin_id: 0,
+        })
+      expect(res.status).toEqual(500);
+    })
+  })
+
+  describe('GET ROUTE /:ID', () => {
+    it('should return 200 on success', async () => {
+      await getOrganizationById();
+      const res = await request(server)
+        .get(ORGS_API_URL + '/:id')
+        .send(req.params.id)
+      expect(res.status).toEqual(200);
+    })
+    it('should return 404 on fail (org does not exist)', async () => {
+      await getOrganizationById();
+      const res = await request(server)
+        .get(ORGS_API_URL + '/:id')
+        .send(req.params.id) //do i need this?

```

Comment:
this one can be removed - it's being passed automatically when you hit the endpoint, e.g. `/api/organizations/1` - then it extracts "1" as the ID.